scalableminds/webknossos
# Cleanup annotations which are in initializing state and older than 1 hour

- create cleanup job for annotations in initializing state and older than 1 hour

### Steps to test:
- create an annotation with state `Initializing` and created < (now() - interval '1 hour')
- reduce the rate of the cleanup service to 30 secs
- wait for log output and confirm that annotation is deleted 

### Issues:
- fixes #2703 

------
